source/custom/nixnet_converters.h

```diff
@@ -3,6 +3,7 @@
 #include <nixnet.h>
 #include <nixnet.pb.h>
 
+#include "server/converters.h"
```

Comment:
note: This include is required for correctness, to ensure that template instantiations for convert methods here works.